### DISCUSSION TITLE
Docker support?

### DISCUSSION THREAD

**Discussion by dschwarz2011** [afourmy/eNMS#371] [General]
Hello, I was wondering about docker support? I saw in previous branches there were dockerfiles but is this deprecated now?

**Comment by afourmy** [comment]
Yes, there used to be a dockerfile but it is deprecated.
eNMS is a standard flask app, it is easy to create a dockerfile for it.

**Reply by dschwarz2011** [reply]
I was receiving an import app error below. Could you post a working dockerfile?


root@netmgr101:/etc/eNMS# docker run -p 5000:5000 -it enms:latest
 * Serving Flask app 'app.py' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
Usage: flask run [OPTIONS]
Try 'flask run --help' for help.

Error: Could not import 'app'.

**Reply by afourmy** [reply]
I don't have one, I haven't used docker in a long time.

**Reply by davama** [reply]
Would be great if there was a good "current" dockerfile to deploy/test quickly :bow:

**Reply by SaneiSaya** [reply]
I converted eNMS to docker and it has instructions.  https://github.com/SaneiSaya/docker-eNMS
